gui/wxpython/lmgr/toolbars.py

```diff
@@ -130,22 +130,22 @@ def _toolbarData(self):
                     self.parent.OnAddMaps,
                 ),
                 (
-                    ("addrast", _("Add raster")),
+                    ("addrast", _("Add raster map")),
                     icons["addRast"],
                     self.parent.OnAddRaster,
                 ),
                 (
-                    ("rastmisc", _("Add various raster")),
+                    ("rastmisc", _("Add various raster map")),
```

Comment:
As "various" implies plural nouns afterwards, I'd use `maps`. Same below.

Reply:
Thanks, I've already fixed it.